batch/inference_job.py

```diff
@@ -122,15 +114,18 @@ def launch(self, job_name, config_file, batch_job_queue):
 
         # Prepare to tar up the current directory, excluding any dvc outputs, so it
         # can be shipped to S3
-        dvc_outputs = get_dvc_outputs()
         tarfile_name = f"{job_name}.tar.gz"
         tar = tarfile.open(tarfile_name, "w:gz", dereference=True)
         for p in os.listdir('.'):
             if p == 'COVIDScenarioPipeline':
                 for q in os.listdir('COVIDScenarioPipeline'):
-                    if not (q == 'packrat' or q.startswith('.')):
+                    if not (q == 'packrat' or q == 'sample_data' or q == 'build' or q.startswith('.')):
                         tar.add(os.path.join('COVIDScenarioPipeline', q))
-            elif not (p.startswith(".") or p.endswith("tar.gz") or p in dvc_outputs or p == "batch"):
+                    elif q == 'sample_data':
+                        for r in os.listdir('COVIDScenarioPipeline/sample_data'):
+                            if r != 'united-states-commutes':
+                                tar.add(os.path.join('COVIDScenarioPipeline', 'sample_data', r))
+            elif not (p.startswith(".") or p.endswith("tar.gz") or p in self.outputs):
```

Comment:
to future proof, should we just ignore everything in `.gitignore` and all files returned by `git lfs ls-files`?

Reply:
yep, let me wire it up

Reply:
or i'll do it in a follow-on PR